src/nile/utils/__init__.py

```diff
@@ -116,3 +121,14 @@ def assert_event_emitted(tx_exec_info, from_address, name, data):
         )
         in tx_exec_info.raw_events
     )
+
+
+def get_hash(contract_name, overriding_path=None):
+    """Return the class_hash of a given contract class."""
+    base_path = (
+        overriding_path if overriding_path else (BUILD_DIRECTORY, ABIS_DIRECTORY)
+    )
+    contract_class = compile_starknet_files(
+        files=[f"{base_path[1]}/{contract_name}.cairo"], debug_info=True
+    )
+    return compute_class_hash(contract_class=contract_class, hash_func=pedersen_hash)
```

Comment:
- The .cairo file shouldn't be based on the base_path which is for artifacts.  I don't think we need to compile here, but can expect that contracts have already been compiled (other functions such as `deploy` also expect this).  So we can just read the artifact json.
- For convenience, this should return hex (`compute_class_hash` returns an integer).  The deployments file also uses hex.

```suggestion
def get_hash(contract_name, overriding_path=None):
    """Return the class hash (in hex) of a given contract class."""
    base_path = (
        overriding_path if overriding_path else (BUILD_DIRECTORY, ABIS_DIRECTORY)
    )
    with open(f"{base_path[0]}/{contract_name}.json", "r") as fp:
        contract_class = ContractClass.loads(fp.read())

    return hex(compute_class_hash(contract_class=contract_class, hash_func=pedersen_hash))
```